### PR TITLE
3362 kubernetes install page

### LINKED ISSUE #3362: Beta: Build Kubernetes install page
Copy doc: https://docs.google.com/document/d/1oiD6UUomf2daZitS3UOosujYBXPbiOMzPtXVZBq-ph0/edit

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Add Kubernetes install page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/kubernetes/install>
- See that page matches the [copy doc](https://docs.google.com/document/d/1oiD6UUomf2daZitS3UOosujYBXPbiOMzPtXVZBq-ph0/edit#)

## Issue / Card

Fixes #3362

### CODE REVIEW

**Comment by anasereijo** [comment]
@frankiegranato I would like to talk about this

File: templates/kubernetes/install.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <h2>Install on a public cloud</h2>
+    <p>There are two ways to install Kubernetes in your software defined infrastructure on any major public cloud with conjure-up or Juju.</p>
+    <ul class="p-inline-list">
+      <li class="p-inline-list__item">
+        <a href="https://tutorials.ubuntu.com/tutorial/install-kubernetes-with-conjure-up">Install Kubernetes with conjure-up&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/kubernetes/install.html]: External link?

File: templates/kubernetes/install.html [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+      </li>
+      <li class="p-inline-list__item">
+        <a href="https://jujucharms.com/kubernetes">Install and manage Kubernetes with Juju&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/kubernetes/install.html]: External link

File: templates/kubernetes/install.html [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+      <div class="p-code-snippet">
+        <input class="p-code-snippet__input" value="snap install microk8s --classic --beta" readonly="readonly">
+        <button class="p-code-snippet__action">Copy to clipboard</button>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <ul class="p-inline-list">
+      <li class="p-inline-list__item">
+        <a href="https://microk8s.io/">Visit microk8s.io&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/kubernetes/install.html]: External

File: templates/kubernetes/install.html [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+      </li>
+      <li class="p-inline-list__item">
+        <a href="https://github.com/juju-solutions/microk8s">Find the project on github&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/kubernetes/install.html]: External

File: templates/kubernetes/install.html [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+      <div class="p-code-snippet">
+        <input class="p-code-snippet__input" value="conjure-up kubernetes" readonly="readonly">
+        <button class="p-code-snippet__action">Copy to clipboard</button>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <ul class="p-inline-list">
+      <li class="p-inline-list__item">
+        <a href="https://conjure-up.io/">Use conjure-up to install OpenStack or Kubernetes&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/kubernetes/install.html]: External?

File: templates/kubernetes/install.html [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+      </li>
+      <li class="p-inline-list__item">
+        <a href="https://maas.io/">Find our more about MAAS&nbsp;&rsquo;</a>
```
Comment (anthonydillon) [inline templates/kubernetes/install.html]: External

**Comment by anthonydillon** [comment]
QA:
These links look odd side by side. I wonder if one should be a link or just stack them.
screenshot-2018-6-25 install canonical kubernetes ubuntu

**Comment by anthonydillon** [comment]
There are a few places on this page with the issue above ^

**Comment by steverydz** [comment]
@anthonydillon Fixed all of the above

File: templates/kubernetes/install.html [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <ul class="p-list">
+      <li class="p-list__item">
+        <a class="p-link--external" href="https://conjure-up.io/">Use conjure-up to install OpenStack or Kubernetes</a>
+      </li>
+      <li class="p-list__item">
+        <a class="p-link--external" href="https://maas.io/">Find our more about MAAS</a>
```
Comment (anthonydillon) [inline templates/kubernetes/install.html]: Should be "Find out more about MAAS"

**Comment by steverydz** [comment]
@anthonydillon Fixed

**Comment by anasereijo** [comment]
@steverydz please amend:

install canonical kubernetes ubuntu

**Comment by steverydz** [comment]
@anasereijo Done

**Comment by frankiegranato** [comment]
This all looks great to me. I'm going to ask Carmine to double check this page all makes sense, but it's a +1 from for now. @steverydz please merge this to Beta asap so Carmine can view it.

**Comment by steverydz** [comment]
Done. Just building now - give it 10 minutes and it should be on beta.ubuntu.com

**Pull request merged**